Dockerfile.base

```
@@ -0,0 +1,18 @@
+# ============================================================
+# OpenClaw Base Image — local build recipe
+# Produces an equivalent of ghcr.io/openclaw/openclaw for
+# cold-start / offline builds where GHCR is unreachable.
+# ============================================================
+FROM node:24-bookworm
+
+# Install openclaw globally (--ignore-scripts avoids esbuild/postinstall issues)
+RUN npm install -g openclaw --ignore-scripts
```

Hadolint (2.14.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)